Dockerfile

```
@@ -0,0 +1,24 @@
+# Build
+
+# We use the same image as the runtime base image so that Python
+# version and location matches.
+FROM us-west1-docker.pkg.dev/serverless-runtimes/google-22/runtimes/python312 AS builder
+
+WORKDIR /pip
+COPY requirements.txt .
+# Needed for gcloud builds submit with gcr.io/cloud-builders/docker (default)
+# USER root
+RUN python -m pip install --no-cache-dir --prefix=python-packages -r requirements.txt
+RUN PYTHONPATH=python-packages/lib/python3.12/site-packages python -m pip freeze
```

Hadolint (2.12.0)

[info] Multiple consecutive `RUN` instructions. Consider consolidation.

(DL3059)